web/src/utils/formatting.ts

```diff
@@ -142,6 +142,7 @@ export const getDateTimeFormatOptions = (
   timezone?: string
 ): Intl.DateTimeFormatOptions => {
   switch (timeAverage) {
+    case TimeAverages.HOURLY_72:
     case TimeAverages.HOURLY: {
```

Comment:
Now I'm glad I opted for the switch case instead of if statements 😅

Reply:
💯 Made this super easy, so thanks!